src/mlpack/methods/ann/layer/layer_types.hpp

```diff
@@ -62,6 +62,7 @@
 #include <mlpack/methods/ann/layer/isrlu.hpp>
 #include <mlpack/methods/ann/layer/softshrink.hpp>
 #include <mlpack/methods/ann/layer/radial_basis_function.hpp>
+#include <mlpack/methods/ann/layer/sqnl.hpp>
```

Comment:
In this file you would need to define it like other functions and also add it to `MoreTypes`.